.github/scripts/branded_preview.sh

```diff
@@ -16,13 +16,16 @@ while IFS= read -r version; do
     versions+=("$version")
 done < <(jq -r '.[]' versions.json)
 
-all_urls=(
-   "https://raw.githubusercontent.com/mongodb/openapi/${branch_name:?}/openapi/v2.json"
-   "https://raw.githubusercontent.com/mongodb/openapi/${branch_name:?}/openapi/v2.yaml"
-)
+all_urls=()
 
 # Fetch and append file URLs from each version
 for version in "${versions[@]}"; do
+    if [[ "${version}" == *"private"* ]]; then
+        all_urls+=("https://raw.githubusercontent.com/mongodb/openapi/${branch_name:?}/openapi/v2/private/openapi-${version}.json")
+        all_urls+=("https://raw.githubusercontent.com/mongodb/openapi/${branch_name:?}/openapi/v2/private/openapi-${version}.yaml")
+        continue
+    fi
+
```

Comment:
```suggestion
```

do we want to add private preview though? I was under the impression that we didn't (note: public preview will be automatically added already)

Reply:
In the scope we only says that the access to the private preview spec will be within the mongodb/openapi repo. we don't mention branded_preview. I think is a nice addition since we could just share a link to the selected customers with the branded preview link. With that sad, I am open to remove it too

Reply:
no strong opinion from m side, easy enough to revisit